packages/atclient/3rdparty/cJSON/README.md

```diff
@@ -0,0 +1,5 @@
+
+
+# 3rdparty/cJSON
+
+The files within this directory are from the [cJSON](https://github.com/DaveGamble/cJSON?tab=readme-ov-file#license) project. Their [license](https://github.com/DaveGamble/cJSON?tab=readme-ov-file#license) hereby grants any person to redistribute and/or modify this software under the terms of the MIT license.
```

Comment:
It would be fitting to have another line here with the Copyright statement from the source repo:

> Copyright (c) 2009-2017 Dave Gamble and cJSON contributors

As that makes an explicit statement about the original source and credits the author(s)

Reply:
https://github.com/atsign-foundation/at_c/pull/91/commits/19951129e451f10b8539459c1e16f3132f796c0d